Makefile

```diff
@@ -18,3 +18,8 @@ run: ${binaries}
 test:
 	go clean -testcache
 	go test ./test -v
+
+verify:
+	./hack/verify-gofmt.sh
+	./hack/verify-golint.sh
+	./hack/verify-govet.sh
```

Comment:
I'd prefer we don't run these scripts in favor or just running the tools directly in the Makefile. Also, without the `find` bits either, that doesn't seem necessary when I ran the tools locally.

Reply:
Yes, I agree, but to make it a part of CI we should have something we can fail when code is not properly formatted as per the `goimport` and `golint` fails.

Reply:
Done that.